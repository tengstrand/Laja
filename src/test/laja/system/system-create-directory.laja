
#set (directory = "target/test-output/abc")

#set (directoryExistsBefore = directoryExists(directory))
#createDirectory(directory)
#set (directoryExistsAfter = directoryExists(directory))

#write "target/test-output/system-create-directory.out"
  Directory exists before: {directoryExistsBefore}
  Directory exists after: {directoryExistsAfter}
#end
