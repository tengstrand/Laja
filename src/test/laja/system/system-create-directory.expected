  Directory exists before: false
  Directory exists after: true
